manifests/init.pp

```diff
@@ -71,6 +71,24 @@
         notify  => Service[$servicename],
         content => template('dhcp/redhat/sysconfig-dhcpd'),
```

Comment:
The file doesn't do anything on EL7 and in fact contains a warning that it shouldn't be edited, so there's no need for this file resource.